recipes-bsp/device-tree/device-tree.bbappend

```diff
@@ -56,6 +64,22 @@ do_configure_append_uz7ev-evcc-hdmi () {
 	fi
 }
 
+# For uz7ev-evcc-hdmi-v BSP only
+do_configure_append_uz7ev-evcc-hdmi-v () {
```

Comment:
not needed as uz7ev-evcc-hdmi-v is a uz7ev-evcc-hdmi machine, so it already has the hdmi.dtsi

Reply:
OK, removed